client/lib/plans/index.js

```diff
@@ -262,3 +262,10 @@ export function applyTestFiltersToPlansList( planName ) {
 
 	return filteredPlanConstantObj;
 }
+
+export const isSkipPlansTestEnabled = () => {
+	return isEnabled( 'plans/skip-plans-for-free' ) &&
+		isEnabled( 'plans/personal-plan' ) &&
+		abtest( 'personalPlan' ) === 'show' &&
+		abtest( 'skipPlansLinkForFree' ) === 'skipPlansForFree';
```

Comment:
It does not force existing users to 'show', so users who already have the variant set to 'hide' would still see 'hide'. That wouldn't impact the results of a test because I would only be testing against new users. But it could lead to some UX weirdness like @apeatling pointed out, so I am going to go through and address that. The abtest will be cleared out once @rodrigoi goes through with #6725 so this code will be somewhat short-lived.